examples/amp-document-recommendations.amp.html

```diff
@@ -0,0 +1,146 @@
+</p>
+<p>
+Donec vehicula nisi eget metus blandit, at semper nunc porttitor. Vestibulum sit amet posuere risus, at mattis nibh. Maecenas ultricies scelerisque nibh et feugiat. Praesent mattis, nibh viverra consequat rhoncus, turpis leo venenatis orci, vitae mollis libero magna eget massa. In dapibus, metus sit amet venenatis finibus, lacus metus rhoncus massa, sit amet mattis tortor massa vitae nunc. Mauris a enim sagittis, condimentum tortor vitae, egestas nulla. Ut dictum laoreet sapien non blandit. Etiam fermentum, magna et tincidunt maximus, ex orci sollicitudin felis, ut dapibus orci ipsum quis leo. Nam accumsan tortor sit amet orci gravida, eget dapibus metus dapibus. Fusce congue ultrices dignissim. Duis quis metus in mi pharetra tempus. Etiam dapibus tellus vitae blandit rhoncus. Fusce commodo risus id sapien ultrices vehicula.
+</p>
+<p>
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vitae libero porta nulla iaculis viverra. Vestibulum consectetur scelerisque varius. Nam pulvinar dui a tortor hendrerit, id iaculis elit auctor. Nam dapibus felis in gravida ornare. Nulla varius id mauris sed venenatis. Sed turpis ex, aliquet nec fermentum eget, sollicitudin non est. Suspendisse tincidunt ornare tortor, at vehicula orci aliquam a. Aliquam eleifend odio quis quam dignissim posuere. Proin ac dolor rhoncus, consectetur ipsum non, dictum est. Nam sollicitudin est eu est aliquet eleifend. Duis condimentum, nisl eu finibus auctor, lectus odio fringilla nisi, quis cursus libero magna imperdiet purus. In condimentum vehicula est, nec varius est suscipit vitae. Maecenas ut sapien diam. Vivamus viverra nisl at quam pellentesque posuere. Cras ut nibh non arcu dignissim elementum.
+</p>
+<p>
+Donec vehicula nisi eget metus blandit, at semper nunc porttitor. Vestibulum sit amet posuere risus, at mattis nibh. Maecenas ultricies scelerisque nibh et feugiat. Praesent mattis, nibh viverra consequat rhoncus, turpis leo venenatis orci, vitae mollis libero magna eget massa. In dapibus, metus sit amet venenatis finibus, lacus metus rhoncus massa, sit amet mattis tortor massa vitae nunc. Mauris a enim sagittis, condimentum tortor vitae, egestas nulla. Ut dictum laoreet sapien non blandit. Etiam fermentum, magna et tincidunt maximus, ex orci sollicitudin felis, ut dapibus  orci ipsum quis leo. Nam accumsan tortor sit amet orci gravida, eget dapibus metus dapibus. Fusce congue ultrices dignissim. Duis quis metus in mi pharetra tempus. Etiam dapibus tellus vitae blandit rhoncus. Fusce commodo risus id sapien ultrices vehicula.
+</p>
+  <amp-document-recommendations>
+    <script type='application/json'>
+      {
+        "recommendations": [
+          {
+            "image": "http://localhost:8000/examples/img/hero@1x.jpg",
```

Comment:
Remove `http://localhost:8000` and just have these as absolute paths (`"image": "/examples/img/hero@1x.jpg",` etc.) The host/port can be overridden.

Reply:
Done.